lambdas/indexer/Makefile

```diff
@@ -15,6 +15,8 @@ wheels: check_env
 # TODO: Rename target: https://github.com/DataBiosphere/azul/issues/1654
 deploy: check_branch check_python check_aws config wheels
 	python -m azul.changelog vendor
+	# Ensure all .pyc files are present for a consistent deployment package
```

Comment:
No silent deviations from requested changes, please. 

There are other reasons for wanting to include all pycs and what you wrote puts horse blinders on the reader.

Reply:
You said

> The comment about deterministic pycs is confusing and bears no relevance in this context

I thought I was addressing your comment by removing the mention of the deterministic `.pyc` files. I didn't see this as a silent deviation, but I will be sure to comment if I think I'm going against your request.

I still though it would be helpful to have a comment here explaining _why_ we decided to include the compiled files in the deployment package, since our reason is not at all obvious, so I left that part of the comment in.

> what you wrote puts horse blinders on the reader.

Are you objecting to how I phrased the comment, or are you objecting to including an explanation?

> There are other reasons for wanting to include all pycs

The only reason I was compiling the source files was so that we would have a consistent deployment. What other reasons did you have in mind for doing this?

Reply:
I have made my decision. Please use the version I originally requested:

![image](https://user-images.githubusercontent.com/5143256/81881395-ecb7ba80-9544-11ea-8399-031bc76897b7.png)